package.json

```diff
@@ -1,27 +1,32 @@
 {
-  "name": "wc3maptranslator",
+  "name": "wc3maptranslator-ts",
   "version": "2.0.0",
   "description": "Module to translate between `war3map` and `json` formats for WarCraft III .w3x maps",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",
-    "test": "nyc mocha",
+    "build": "tsc --build",
+    "test": "mocha",
+    "test-nyc": "nyc mocha",
     "test-travis": "./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --include-all-sources true --root ./lib -- -R spec ./test/*"
   },
   "repository": {
     "type": "git",
     "url": "git+https://chiefofgxbxl@github.com/ChiefOfGxBxL/WC3MapTranslator.git"
   },
-  "author": "ChiefOfGxBxL",
+  "author": "ChiefOfGxBxL + rufreakde",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/ChiefOfGxBxL/WC3MapTranslator/issues"
   },
   "homepage": "https://github.com/ChiefOfGxBxL/WC3MapTranslator#readme",
   "dependencies": {
+    "@types/fs-extra": "^8.0.1",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.12.14",
+    "@types/round-to": "^4.0.0",
     "ieee754": "^1.1.8",
     "intn": "^1.0.0",
-    "round-to": "^3.0.0"
+    "round-to": "^4.0.0"
```

Comment:
I checked if the major API would break anything: https://github.com/sindresorhus/round-to/releases

It requires Node >= 8 (which has now been specified in the README), and coincidentally round-to v4 adds TypeScript support 😛